8.frontend-react/crm-frontend/src/screens/Home.jsx

```diff
@@ -1,18 +1,201 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import Header from '../components/header/Header';
 import PageTitle from '../components/pageTitle/PageTitle';
 import '../styles/crmPage.css'
-import '../styles/styles.scss'
-
+import GroupedBar from '../components/groupedBar/GroupedBar';
+import LineChart from '../components/lineChart/LineChart';
+import PieChart from '../components/pieChart/PieChart';
+import CrmApi from '../helpers/CrmApi';
+import DatePicker from '../components/datePicker/DatePicker';
+import MiniProjectsList from '../components/miniProjectsList/MiniProjectsList';
 
 function Home(props) {
+    const [costChartData, setCostChartData] = useState({});
+    const [statusChartData, setStatusChartData] = useState({});
+    const [maxWorkingHours, setMaxWorkingHours] = useState({});
+    const [minWorkingHours, setMinWorkingHours] = useState({});
+    const [leastRecentlyCreatedProjects, setLeastRecentlyCreatedProjects] = useState([]);
+    const crmApi = new CrmApi();
+    const [dateStatus, setDateStatus] = useState("week");
+    const [dateCost, setDateCost] = useState("week");
+    const [dateMinWorking, setMinWorkingDate] = useState("month");
+    const [dateMaxWorking, setMaxWorkingDate] = useState("month");
+    const [dateLeastRecentlyCreatedProject, setDateLeastRecentlyCreatedProject] = useState("month");
+
+    const fetchCost = () => {
+        (async () => {
+            const result = await crmApi.postRequest('/dashboard/projectCostVsEstimated/', {interval: dateCost, limit: 5});
+            if(result){
+                const labels = [];
+                const estimated = {label: 'Estimated Cost', data: []};
+                const actual = {label: 'Actual cost', data: []};
+                for(let item of result){
+                    labels.push(item.user_name);
+                    estimated.data.push(item.estimated_cost);
+                    actual.data.push(item.total_cost);
+                }
+                setCostChartData({labels: labels, datasets: [estimated, actual]})
+            }
+          })();
+    } 
+
+    useEffect(()=>{
+        fetchCost();
+        const interval =  setInterval(fetchCost, 20 * 1000);
+        return () => clearInterval(interval);
+    }, [dateCost])
+
+
+    const fetchStatus = () => {
+        (async () => {
```

Comment:
why is the double function ? 🤔 
it should behave the same if you make "fetchStatus" async, and run the content in it